where-to-travel/src/main/webapp/script.js

```diff
@@ -1,141 +1,188 @@
-//This is map stylings for the GMap api
+// This is map stylings for the GMap api
 const mapStyles = [
   {
-    "featureType": "landscape",
-    "stylers": [
+    featureType: 'landscape',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   },
   {
-    "featureType": "poi",
-    "elementType": "labels.icon",
-    "stylers": [
+    featureType: 'poi',
+    elementType: 'labels.icon',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   },
   {
-    "featureType": "transit",
-    "stylers": [
+    featureType: 'transit',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   }
 ];
-//End map stylings
+// End map stylings
 
-const submitId = "submit-id";
-const hoursId = "hrs";
-const minutesId = "mnts";
+
+// Thresholds for termination of search algorithm
+const placesThreshold = 30;
```

Comment:
Prefer constant case for constant names: https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
fixed!